src/main/java/school/lemon/changerequest/java/extendedinteger/ExtendedInteger.java

```diff
@@ -4,116 +4,116 @@
  * Integer representation with some additional features
  */
 public class ExtendedInteger {
+    private int value;
 
     public ExtendedInteger(int value) {
-        //TODO: implement me
+        this.value = value;
     }
 
-    /**
-     * Check whether specified value is even
-     * @param value to check
-     * @return true if value is even, false - otherwise
-     */
+
     public static boolean isEven(int value) {
-        //TODO: implement me
-        return false;
+        return ((value & 1) == 0);
     }
 
-    /**
-     * Check whether specified value is odd
-     * @param value to check
-     * @return true if value is odd, false - otherwise
-     */
     public static boolean isOdd(int value) {
-        //TODO: implement me
-        return false;
+        return (value % 2 != 0);
     }
 
-    /**
-     * Check whether specified value is prime
-     * @param value to check
-     * @return true if value is prime, false - otherwise
-     */
     public static boolean isPrime(int value) {
-        //TODO: implement me
-        return false;
+        if (value == 2) return true;
+        if (value < 2 || value % 2 == 0) return false;
+        for (int i = 2; i < value; i++) {
```

Comment:
If you have already verified that `2` will return `true` and everything that could be divided by `2` will return false - you can start with `3` in you `for`-loop and iterate with `i+=2`, cause you don't need anything that could be divided by `2`.